packages/edit-site/src/components/list/style.scss

```diff
@@ -1,3 +1,10 @@
+.edit-site-list {
+	.interface-interface-skeleton__editor {
```

Comment:
Should be the responsibility of each page (here list page) to define the radius and background of the frame or should this be something that is defined by default in the upper level (frame_container)

Reply:
IMO our strongest design statement so far is to keep the frame uniform across everywhere it's being used. There are some good arguments that we can potentially scale it in some contexts, but this is something I'd think us to explore separately, and in either case, I wouldn't think the radius should ever change.